core/base-service/base.js

```diff
@@ -1,8 +1,12 @@
 'use strict'
 
 // See available emoji at http://emoji.muan.co/
+const decamelize = require('decamelize')
 const emojic = require('emojic')
 const Joi = require('joi')
+// Ideally `validateMetricName()` would be in the public interface.
+// https://github.com/siimon/prom-client/pull/246
```

Comment:
Looks like your PR has already been approved so hopefully that will get merged in before too long 👍

Reply:
Though… i've just removed it 😀